PLEP-0008.rst

```diff
@@ -0,0 +1,166 @@
+PLEP-0008 – Calendar Versioning
+===============================
+
++-------------------+---------------------------------------------+
+| PLEP              | number                                      |
++===================+=============================================+
+| title             | PlasmaPy Versioning and Releases            |
++-------------------+---------------------------------------------+
+| author(s)         | Nick Murphy                                 |
++-------------------+---------------------------------------------+
+| contact email     | namurphy@cfa.harvard.edu                    |
++-------------------+---------------------------------------------+
+| date created      | 2022-11-22                                  |
++-------------------+---------------------------------------------+
+| date last revised | 2022-11-29                                  |
++-------------------+---------------------------------------------+
+| type              | process                                     |
++-------------------+---------------------------------------------+
+| status            | discussion                                  |
++-------------------+---------------------------------------------+
+| DOI               |                                             |
+|                   |                                             |
++-------------------+---------------------------------------------+
+
+Brief Description
+-----------------
+
+PlasmaPy will use `calendar versioning`_ starting with the first release
+of 2023.
+
+.. note::
+
+    This PLEP supersedes |PLEP 5|, which is no longer in effect.
+
+Versioning Specification
+------------------------
+
+PlasmaPy final releases will use `calendar versioning`_ of the
+form ``YYYY.0M.MICRO``, where:
```

Comment:
Leading zeros will be removed in accordance with [PEP440](https://peps.python.org/pep-0440/#integer-normalization).  Basically these are normalized in accordance with `int()`.